cmd/vm.go

```diff
@@ -30,6 +30,8 @@ var vmCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(vmCmd)
 	vmCmd.Flags().StringVarP(&vmOptions.IP, "ip", "i", "none", "External IP of kvmsoperator")
-	vmCmd.Flags().StringVarP(&vmOptions.VMName, "vmname", "n", "", "Name of configured vm")
+	vmCmd.Flags().StringVarP(&vmOptions.Port, "port", "p", "none", "Port of kvmsoperator")
+	vmCmd.Flags().StringVarP(&vmOptions.Namespace, "ns", "n", "none", "Namespace of kvmsoperator")
```

Comment:
We should default to `kube-system` namespace if no flag input is provided.

Reply:
Taken care of the same. Default will be kube-system.